listener/upload.go

```diff
@@ -97,9 +97,10 @@ func HandleUpload(event HostEvent) error {
 			return nil
 		}
 	}
-
+	t := time.Now().Format(base.Rfc3339NoTz)
```

Comment:
This should be done also (and mainly) in vmaas_sync when sending recalc messages.

Reply:
Done, thanks